source/browseMode.py

```diff
@@ -1778,7 +1795,10 @@ def doResult(result):
 			obj = container
 		return doResult(False)
 
-	def _get_documentConstantIdentifier(self):
+	documentConstantIdentifier: Optional[str]

```

Comment:
Auto property docstring
```suggestion
	documentConstantIdentifier: Optional[str]
	""" Typing information for auto-property: _get_documentConstantIdentifier"""
```